django_react_proj/users-fe/src/components/TaskButtonModal/TaskButtonModal.js

```diff
@@ -1,49 +1,34 @@
-import React, { useState, Fragment } from "react";
+import React, {useState} from "react";
 import { Modal, ModalHeader, Button, ModalFooter } from "reactstrap";
-import styles from './TaskButtonModal.module.css';
+import styles from "./TaskButtonModal.module.css";
 
 const TaskButton = (props) => {
-  const [modalFlag, setModalFlag] = useState(
-        false
-  )
-
-  const [taskFlag, setTaskFlag] = useState(
-    false
-  )
+  const [modalFlag, setModalFlag] = useState(false);
 
   const toggle = () => {
-    setModalFlag(previous => (
-       !previous
-     ));
+    setModalFlag((previous) => !previous);
   };
-  
-  const handlerDeleteUser = async id => {
+
+  const handlerDeleteUser = async (id) => {
     try {
-      await props.deleteUser(id)
-      toggle() 
+      await props.deleteUser(id);
+      toggle();
     } catch (error) {
-      console.log(error)
-    } 
-   };
-
-  const toggleTaskFlag = () => {
-    setTaskFlag(previous => (
-        !previous
-      ));
+      console.log(error);
+    }
   };
-   
+
   return (
     <>
       <Button
-        color = {taskFlag ? 'success': 'danger'}
-        onClick = {() => toggleTaskFlag()}>
+        color = {'primary'}
+        onClick={() => props.ontoggleTaskFlag()}
```

Comment:
If you don't need to pass any arguments to this function this can be simply `={props.onToggleTaskFlag}`.